packages/bundle-source/src/index.js

```diff
@@ -276,15 +276,22 @@ async function bundleZipBase64(startFilename, options = {}, powers = {}) {
       sourceMapJobs.add(writeSourceMap(sourceMap, sourceDescriptor));
     },
   });
+  assert(sha512);
   await Promise.all(sourceMapJobs);
   const endoZipBase64 = encodeBase64(bytes);
   return harden({
-    moduleFormat: 'endoZipBase64',
+    moduleFormat: /** @type {const} */ ('endoZipBase64'),
```

Comment:
I wish we had enums. Consider having something up top like,
```
export const ModuleFormat = /** @type {const} */ ({
  getExport: 'getExport',
  nestedEvaluate: 'nestedEvaluate',
  endoZipBase64: 'endoZipBase64',
});

const DEFAULT_MODULE_FORMAT = ModuleFormat.endoZipBase64;
const DEFAULT_FILE_PREFIX = '/bundled-source/...';
const SUPPORTED_FORMATS = Object.values(ModuleFormat);
```
then the string wouldn't need a const cast when used,
```suggestion
    moduleFormat: ModuleFormat.endoZipBase64,
```

I'm on the fence about it myself since we don't have enums and TS will detect string literal errors. But it does make it read a little easier

Reply:
I'm not entirely sold on it either.  So many of our callers traffic in string moduleFormats, and I don't want to add abstractions unless they're motivated by the callers.